src/routes/jira/connect/enterprise/jira-connect-enterprise-post.ts

```diff
@@ -69,9 +69,9 @@ export const JiraConnectEnterprisePost = async (
 	// inside the handler
 	const TIMEOUT_PERIOD_MS = parseInt(process.env.JIRA_CONNECT_ENTERPRISE_POST_TIMEOUT_MSEC || "30000");
 
-	const gheServerURL = req.body.gheServerURL?.trim();
-	const apiKeyHeaderName = req.body.apiKeyHeaderName?.trim();
-	const apiKeyValue = req.body.apiKeyValue?.trim();
+	const gheServerURL: string = req.body.gheServerURL?.trim();
```

Comment:
Hmm, curious why TS is not complaining... `req.body.gheServerURL?.trim()` could be `undefined`, right?

Reply:
Correct updated to address.  TS can't work it out because it is an any object.  It will complain here once we enable unsafe call to any rules later.